.changeset/real-weeks-check.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/identity-apps-core": major
```

Comment:
Shall we make this patch since this is not a major release?

Reply:
Fixed with 6957fe72b0ae8e942c028f5cd75569b27120bf4c